taskcluster/.build.yml

```diff
@@ -23,4 +23,4 @@ build:
     tests_cmdline: ''
   convert_graphdef: ''
   benchmark_model_bin: ''
-  tensorflow_git_desc: 'TensorFlow: v1.14.0-14-g1aad02a'
+  tensorflow_git_desc: 'TensorFlow: v1.12.0-15245-gc40bd58'
```

Comment:
What's up with this? Forgot to push tags for the TF PR?

Reply:
Yeah on my fork, it should be okay once on our repo :)